### DISCUSSION TITLE
Resize shared layout files to own screen size + sharing of layouts on website?

### DISCUSSION THREAD

**Discussion by ghost** [f1multiviewer/issue-tracker#108] [Feature requests]
A feature which allows you to "resize" the layouts file to your own screen size, and maybe even a place to share them on the website (maybe a link to the layout file included in on the showcase page)

**Comment by Robertsmania** [comment]
Aspect ratios may make that more challenging than it seems.  My system upstairs is 1920x1080/16:9 but my laptop is 2880x1800/16:10.  So unless you do old school letter/pillar-box tricks to make it fit its going to be off
